apps/docs/content/components/RiverAccordion.mdx

```diff
@@ -0,0 +1,193 @@
+    <RiverAccordion.Heading>Heading 2</RiverAccordion.Heading>
+    <RiverAccordion.Content>
+      <Text as="p" variant="muted" weight="medium" size="200">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+        ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+        elementum.
+      </Text>
+      <Link href="#" variant="accent">
+        Call to action
+      </Link>
+    </RiverAccordion.Content>
+    <RiverAccordion.Visual>
+      <img
+        src="https://placehold.co/600x400/AF7AC5/ffffff?text=2"
+        alt="Visual content for item 2"
+      />
+    </RiverAccordion.Visual>
+  </RiverAccordion.Item>
+
+  <RiverAccordion.Item>
+    <RiverAccordion.Heading>Heading 3</RiverAccordion.Heading>
+    <RiverAccordion.Content>
+      <Text as="p" variant="muted" weight="medium" size="200">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+        ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+        elementum.
+      </Text>
+      <Link href="#" variant="accent">
+        Call to action
+      </Link>
+    </RiverAccordion.Content>
+    <RiverAccordion.Visual>
+      <img
+        src="https://placehold.co/600x400/48C9B0/ffffff?text=3"
+        alt="Visual content for item 3"
+      />
+    </RiverAccordion.Visual>
+  </RiverAccordion.Item>
+</RiverAccordion>
+```
+
+## Component props
+
+### RiverAccordion <Label>Required</Label>
+
+| Name       | Type                                                        |  Default  | Description                                                            |
+| :--------- | :---------------------------------------------------------- | :-------: | :--------------------------------------------------------------------- |
+| `children` | `React.ReactElement[]`                                      |           | RiverAccordion.Item components                                         |
+| `align`    | <PropTableValues values={['start', 'end']} addLineBreaks /> | `'start'` | Controls the position of the visuals relative to the accordion content |
+
+`RiverAccordion` extends the HTML `div` element and supports all `div` props.
+
+### RiverAccordion.Item <Label>Required</Label>
+
+| Name       | Type                   | Default | Description                                          |
+| :--------- | :--------------------- | :-----: | :--------------------------------------------------- |
+| `children` | `React.ReactElement[]` |         | Must include Heading, Content, and Visual components |
+
+`RiverAccordion.Item` extends the HTML `div` element and supports all `div` props.
+
+### RiverAccordion.Heading <Label>Required</Label>
+
+| Name       | Type     | Default | Description  |
+| :--------- | :------- | :-----: | :----------- |
+| `children` | `string` |         | Heading text |
+
+`RiverAccordion.Heading` extends the [`Heading`](/components/Stack) component and supports all `Heading` props.
```

Comment:
```suggestion
`RiverAccordion.Heading` extends the [`Heading`](/components/heading) component and supports all `Heading` props.
```

Reply:
Thank you, I swear I spotted and fixed this once already, but clearly not! 😅